Sources/LanguageServerProtocol-Server/Server.ProtocolHandler.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import JSONRPC
+import LanguageServerProtocol
+import Logging
+
+public protocol ProtocolHandler {
+  var connection: JSONRPCClientConnection { get }
```

Comment:
This protocol gives me pause. A synchronous getter can be very problematic for an actor. I haven't yet looked closely enough to be sure, but it seems like it may be possible to make this `{ get async }`.

Reply:
This has been refactored and the protocol is not there anymore, so not sure if it is still something that needs fixing in the updated changeset. This is a bit beyond my swift knowledge atm!